webapp/components/Comment/Comment.vue

```diff
@@ -54,6 +54,15 @@
         </button>
       </div>
       <ds-space margin-bottom="small" />
+      <base-button
+        :title="this.$t('post.comment.reply')"
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-09T07:56:32Z" title="Thursday, January 9th 2020, 8:56:32 am +01:00">Jan 9, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -55,6 +55,13 @@
       </div>
       <ds-space margin-bottom="small" />
     </ds-card>
+    <ds-button
+      v-bind:title="answered"
```


`:title` dies it and we prefer.
`"answered“` see later … 👇🏼

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-15T06:41:16Z" title="Wednesday, January 15th 2020, 7:41:16 am +01:00">Jan 15, 2020</time>_

---


ok